frontend/tests/home/homeSpecShallow.js

```diff
@@ -0,0 +1,52 @@
+import test from 'blue-tape';
+import sinon from 'sinon';
+import React from 'react/addons'; // React is required here to render Tasks
+import createComponent from '../utils/create-component';
+import Home from '../../src/components/home/home.jsx';
+
+let onLogin = () => {
+  console.log('onlogin was called');
+  return true;
+};
+
+let lock = {
+  show(callback) {
+    callback();
+  }
+};
+
+let home = createComponent(Home, {lock: lock});
```

Comment:
Lines 7–18 were not even used in this test; they are for rendering of an instance of a component, while I was more interested in testing the component's methods.